### PR TITLE
FFM-7811: adds ff server relnotes 1.1054.0

### PR DESCRIPTION
**Pull request by StrawberryPad**
[**Link to preview**](https://645e434968e6bc0b998e58e7--harness-developer.netlify.app/release-notes/feature-flags)

### CODE REVIEW

**Review by jcox250 — DISMISSED** [review]

**Comment by StrawberryPad** [comment]
Thanks @jcox250 !

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -38,9 +38,17 @@ This release does not include early access features.
...
-* Fixed an issue in the onboarding flow where the flag validation did not work as expected. (FFM-7534)
+* Previously, re-enabling Git Experience did not trigger an immediate Git sync. With this change, flags are synchronized as soon as Git Experience is re-enabled. (FFM-7670)
+
+* Some Git sync operations were failing if there was a large volume of flags and environments. This fix increased the transaction time-out for Git sync calls to allow processing of a large number of the flags. (FFM-7638)
+
+* Policy checks weren't being carried out on flag rules added from the Target Group details page. This issue has been fixed. (FFM-7607)(FFM-7606)
+
+* Previously, Feature Flag permissions and roles assigned to users or user groups in Access Control were applied at the account and project levels, but not at the organization level. With this change, roles and permissions assigned at the organization level are now honored. (FFM-7376)
+
+* When trying to delete a flag that is a prerequisite to another flag, a new explanatory message is now displayed for why this cannot be done: *Cannot delete flag which is a prerequisite for other flags*. (FFM-5105)
```
Comment (douglas-j-bothwell) [inline release-notes/feature-flags.md:51]:
```suggestion
* When you try to delete a flag that is a prerequisite to another flag, the UI now shows a message that explains why this cannot be done: *Cannot delete flag which is a prerequisite for other flags*. (FFM-5105)
```

**Review by douglas-j-bothwell — DISMISSED** [review]
One minor sug but otherwise looks good!

**Review by davejohnston — APPROVED** [review]
LGTM

**Pull request merged**